### PR TITLE
[AST-175] Update refractor to fix prismjs vulnerabilities

### PR DESCRIPTION
**Pull request by lucianomlima**
Update refractor for astro-native@next

- fix [Cross-Site Scripting](https://github.com/magnetis/astro-native/security/dependabot/1)
- fix [Denial of service](https://github.com/magnetis/astro-native/security/dependabot/6)
- fix [ReDoS in Prism](https://github.com/magnetis/astro-native/security/dependabot/17)
- fix [ReDoS in prismjs](https://github.com/magnetis/astro-native/security/dependabot/24)
- fix [Cross-site Scripting](https://github.com/magnetis/astro-native/security/dependabot/36)
- update caniuse-lite database to fix warning during tests

### CODE REVIEW

**Review by albertozaranza — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -95,6 +95,15 @@
...
   },
+  "resolutions": {
+    "refractor": "^3.6.0"
+  },
+  "husky": {
+    "hooks": {
+      "pre-commit": "lint-staged",
+      "commit-msg": "commitlint -E HUSKY_GIT_PARAMS"
+    }
+  },
```
Comment (LopesAlysson) [inline package.json:106]: This change is not related to the commit title or body

Comment (lucianomlima) [reply package.json:106]: It's actually in the body. Take another look at the last few lines. 😉

**Review by LopesAlysson — APPROVED** [review]
